bot.py

```diff
@@ -212,7 +212,8 @@ def callback_handler(bot: Bot, update: Update):
 
 def subscribe_handler(bot: Bot, update: Update, args: List[str], job_queue: JobQueue):
     filter_text = demojize("".join(args))
-    if user_db.is_subscriber(update.message.chat_id):
+    isRefreshed = True if user_db.menu_filter_of(update.message.chat_id) == filter_text else False
```

Comment:
snake-case, please :smile:

Reply:
btw: `True if x else False` is the same as `x`

Reply:
Awww sneaky snake